components/profile/dashboard/total-contribution.js

```diff
@@ -0,0 +1,83 @@
+import { 
+  Avatar,
+  Box,
+  Card,
+  CardContent,
+  Grid,
+  LinearProgress,
+  Typography
+} from '@mui/material';
+import ControlPointIcon from '@mui/icons-material/ControlPoint';
+import {ContributionConst} from '../../../shared/contribution-const';
+
+const TotalContribution = (props) => {
+  const {totalCP} = props;
+  var currentTier = 'Untiered';
+  var nextTierCP = 0;
+  for (const tierName in ContributionConst.tier) {
+    if (totalCP < ContributionConst.tier[tierName]) {
+      nextTierCP = ContributionConst.tier[tierName];
+      break;
+    }
+    else
+      currentTier = tierName;
+  }
+  return (
+    <Card {...props}>
+      <CardContent>
+        <Grid
+          container
+          spacing={3}
+          sx={{ justifyContent: 'space-between' }}
+        >
+          <Grid item>
+            <Typography
+              color="textSecondary"
+              gutterBottom
+              variant="overline"
+            >
+              TOTAL CONTRIBUTION
+            </Typography>
+            <Typography
+              color="textPrimary"
+              variant="h4"
+            >
+              {totalCP} Pts
+            </Typography>
+            <Typography
+              color="textSecondary"
+              variant="subtitle1"
+            >
+              {currentTier} Tier
+            </Typography>
+          </Grid>
+          {/* <Grid item>
+            <Avatar
+              sx={{
+                backgroundColor: 'success.main',
+                height: 56,
+                width: 56
+              }}
+            >
+              <ControlPointIcon />
+            </Avatar>
+          </Grid> */}
```

Comment:
same with comment in `monthly-contribution.js` file

Reply:
same with the comment above